tests/utils.py

```diff
@@ -443,19 +467,47 @@ def __file_comparer(x: str, y: str) -> bool:
     return sha256sum(x) == sha256sum(y)
 
 
+def __img_comparer(img1: PIL.Image.Image, img2: PIL.Image.Image) -> bool:
```

Comment:
The existing tests just use `__equality_comparer` to compare images (in general `x == y` should do the right thing if `x` and `y` are PIL images). Is there some reason why equality comparison isn't adequate for the new tests? If so please add a comment to explain.